interview-questions.md

```diff
@@ -7,44 +7,78 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: To fix this mistake see method below. The name of the foreign key would be has_many students and is on the student model.
 
   Researched answer:
 
+  To correct the mistake: Step 1: run the command 'rails generate migration', add a new column to the table. Step 2: add_column :students, :cohort_id, :integer' Step 3: lastly, in terminal, run 'rails db:migrate'.
 
 
 2. Which RESTful routes must always be passed params? Why?
```

Comment:
https://github.com/learn-academy-2022-charlie/Syllabus/blob/main/rails/restful-routes-crud.md
Look at the paths that require an id